docs/products/clickhouse/concepts/supported-table-engines.rst

```diff
@@ -0,0 +1,63 @@
+Supported table engines
+=======================
+
+This article lists the table engines that are currently supported in Aiven for ClickHouse.
+
+ClickHouse supports several table engines that define the storage parameters, supported queries, and other aspects of the data tables. For details on each table engine, see the `ClickHouse documentation <https://clickhouse.com/docs/en/engines/table-engines/>`_.
+
+Aiven for ClickHouse supports the following table engines:
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Engine
+    - Engine family
+  * - ``AggregatingMergeTree`` (remapped)
```

Comment:
do we have somewhere info what "remapped" means? I can also clarify with with the ClickHouse team

Reply:
Yep, that's now added as a note ;)